packages/react-components/src/molecules/ModalEditHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { Headline3, Button, Link } from '../atoms';
+import { crossIcon } from '../icons';
+import { tabletScreen } from '../pixels';
+import { noop } from '../utils';
+
+const headerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    flexDirection: 'row-reverse',
+    justifyContent: 'space-between',
+  },
+});
+
+const controlsContainerStyles = css({
+  display: 'grid',
+  gridTemplateColumns: 'min-content min-content',
+  alignItems: 'flex-end',
+  justifyContent: 'flex-end',
```

Comment:
```suggestion
  alignItems: 'end',
```
`flex-end` in a grid seems weird, and justify-content isn't needed I think?